src/Backend/Modules/Pages/Layout/Templates/Offline.html.twig

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Offline page</title>
+  <meta name="description" content="Offline page of this awesome website">
+  <meta name="author" content="Sarah Meysman">
+
+  <!--<link href="css/bootstrap.min.css" rel="stylesheet" />-->
+  <link href="/src/Frontend/Themes/Fork/Core/Layout/Css/screen.css?m=1593680880" rel="stylesheet" />
```

Comment:
You should load the css in a proper way, you hardcoded the theme and the timestamp